graphing/src/cloudevolution/views.py

```diff
@@ -7,6 +7,8 @@
 import itertools
 import os
 import time
+import math
+
```

Comment:
Please remove the extra white space added throughout the PR. `sed` would be a good way to do it, or manually.